lib/assets/javascripts/turbograft/turbolinks.coffee

```diff
@@ -56,14 +56,204 @@ removeNode = (node) ->
   removedNode = node.parentNode.removeChild(node)
   triggerEvent('page:after-node-removed', removedNode)
 
+updateHeadOrRefresh = (url, upstreamDocument, callback) ->
+  activeAssets = extractTrackedAssets(document)
+  upstreamAssets = extractTrackedAssets(upstreamDocument)
+  updateLinkTags(activeAssets, upstreamAssets)
+  updateScriptTagsOrRefresh(url, activeAssets, upstreamAssets, callback)
+
+extractTrackedAssets = (doc) ->
+  for node in doc.head.childNodes when node.getAttribute?('data-turbolinks-track')?
+    node
+
+insertScriptTask = (scriptNode) ->
+  newNode = document.createElement('SCRIPT')
+  newNode.setAttribute attr.name, attr.value for attr in scriptNode.attributes
+  newNode.setAttribute 'async', false
```

Comment:
Isn't order guaranteed by `asyncSeries`?  That should make `async`/`defer`/neither irrelevant, no?